.github/workflows/ci.yml

```
@@ -2,262 +2,337 @@ name: CI/CD Pipeline
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        cache: 'npm'
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Run npm audit
+      run: npm audit --audit-level=moderate
+      continue-on-error: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)